### ISSUE TITLE
Feature request: Shorter wording for PAX miner success notification in slack. 

### ISSUE THREAD

**Issue by georgeinmexico** [F3Nation-Community/PAXminer#23]
I live in fear of making Paxminer unhappy, because I think it sends me one slack notification per hour until I fix a badly formatted backblast. If I'm away from my computer for a day, that can mean 24 notifications.

When Slack sends me an iphone or applewatch notification (about anything), I see the first 100 characters of the message. Currently, this is the as much of the paxminer slack notification as I can read from my apple watch:

_Hey Sparrow - I see a backblast you posted on 2024-04-03 at [#ao_dark_room] Here's what happened..._

I can't tell if this a good omen or bad omen. I continue living in fear. 

This would be my preferred paxminer notification:

Paxminer: Success! I imported your backblast for 2024-03-20 at [#ao_dark_room]. I see you had 9 PAX and 0 FNGs.

or 

Paxminer failed to import your latest backblast...

In conclusion. I love paxminer. It works great. Our region (F3 wheaton) uses it religiously and we use the paxminer stats as the final authority in some very competitive competitions. keep of the good work, Beaker!

**Comment by farrellw** [comment]
@georgeinmexico this request makes sense to me. 

Your life should have gotten better when we adjusted the PAXMiner code to only send failure messages up to twice a day. However rearranging the phrasing makes perfect sense. Thanks for bringing it to our attention.

@srschaecher here is an MR that addresses this request https://github.com/F3Nation-Community/PAXminer/pull/34

**Comment by farrellw** [comment]
@georgeinmexico This suggestion has been merged and deployed

**Comment by georgeinmexico** [comment]
Thank you!!!
